### PR TITLE
Bitcoin Core 25.0

### PR DESCRIPTION
**Pull request by fanquake**
Add release pages for Bitcoin Core v25.0.

### CODE REVIEW

File: _posts/en/posts/2023-05-21-release-25.0.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  Bitcoin Core 25.0 is now available.
+---
+Bitcoin Core version 25.0 is now available for [download][download
+page].  See the [release notes][] for more information about the many
+new features and bug fixes in this release.
+
+If you have any questions, please stop by the #bitcoin IRC chatroom
+([IRC][irc], [web][web irc]) and we’ll do our best to help you.
+
+[release notes]: /en/releases/23.1/
```
Comment (odarboe) [inline _posts/en/posts/2023-05-21-release-25.0.md:25]: Should this be 25.0 or you want to maintain 23.1?
`/en/releases/23.1/`

Comment (fanquake) [reply _posts/en/posts/2023-05-21-release-25.0.md]: Thanks. Fixed.

**Comment by fanquake** [comment]
Rebased  on master. Moved the dates forward a few days, and removed `merge-script` from the release notes.

**Comment by fanquake** [comment]
Added the torrent link. Re-imported the release-notes to include changes from https://github.com/bitcoin/bitcoin/pull/27751. Binaries are now available.

**Comment by fanquake** [comment]
cc @stickies-v

File: _includes/posts/maintenance-table.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 | 22.0 | 2021-09-13 | 2022-11-24  | 2023-04-01 |
```
Comment (stickies-v) [inline _includes/posts/maintenance-table.md:1]: I think `22.0` can be removed now?

File: _releases/25.0.md [hunk 3]
```diff
@@ -0,0 +1,368 @@
+---
+title: Bitcoin Core 25.0
+id: en-release-25.0
+name: release-25.0
+permalink: /en/releases/25.0/
+excerpt: Bitcoin Core version 25.0 is now available
+date: 2023-05-23
```
Comment (stickies-v) [inline _releases/25.0.md:7]: nit
```suggestion
date: 2023-05-26
```

**Review by stickies-v — APPROVED** [review]
ACK 92c5951d9ba0c7342c36779afcc89524f6ca4eec

Release notes match https://github.com/bitcoin/bitcoin/pull/27751/files. Also verified the shasums and signatures of the binaries in the magnet link.

File: _includes/posts/maintenance-table.md (hunk 2 above)
Comment (fanquake) [reply _includes/posts/maintenance-table.md:1]: Going to leave it as-is for a month or two.

File: _releases/25.0.md (hunk 3 above)
Comment (fanquake) [reply _releases/25.0.md]: Changed.

**Review by stickies-v — APPROVED** [review]
re-ACK [2108220](https://github.com/bitcoin-core/bitcoincore.org/commit/2108220c916a99f2409bb2ada6fbbecfdbed7f65) - only the date changed.

File: _releases/25.0.md [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+## Use the same number of elements as decimal places, e.g. "0.1.2 => [0,
+## 1, 2]" versus "1.2 => [1, 2]"
+release: [25, 0]
+
+## Optional magnet link.  To get it, open the torrent in a good BitTorrent client
+## and View Details, or install the transmission-cli Debian/Ubuntu package
+## and run: transmission-show -m <torrent file>
+#
+## Link should be enclosed in quotes and start with: "magnet:?
+optional_magnetlink: "magnet:?xt=urn:btih:092358777175c4306602f9b1b523738df4b4610b&dn=bitcoin-core-25.0&tr=udp%3A%2F%2Ftracker.openbittorrent.com%3A80&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969%2Fannounce&tr=udp%3A%2F%2Fexplodie.org%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.torrent.eu.org%3A451%2Fannounce&tr=udp%3A%2F%2Ftracker.bitcoin.sprovoost.nl%3A6969&ws=http%3A%2F%2Fbitcoincore.org%2Fbin%2F"
```
Comment (Sjors) [inline _releases/25.0.md:20]: Added `092358777175c4306602f9b1b523738df4b4610b` to my tracker whitelist, but haven't verified the binary.

**Review by Dezzj — APPROVED** [review]
Approved

**Pull request merged**